### PR TITLE
Issue#2-creating tables through classes

### PR DESCRIPTION
**Pull request by Nikhitha-wsu**
Adding patient_generalinfo and patient_healthinfo tables to the database.

### CODE REVIEW

**Comment by SaiPriya-WSU** [comment]
I have checked models.py necessary classes were created please add fields like blood pressure, height, weight, doctor name, date of diagnosis

**Comment by Nikhitha-wsu** [comment]
> I have checked models.py, necessary classes were created. Please add fields like blood pressure, height, weight, doctor name, date of diagnosis in patient_healthinfo class.

I have added all the necessary columns as suggested.

File: PatientInfoapp/models.py
Comment (Prathyusha-WSU) [inline PatientInfoapp/models.py:1]: Good Job! I have reviewed the models.py class. All the required tables and columns were added. Also the initial data is populated.

Comment (SaiPriya-WSU) [reply PatientInfoapp/models.py:1]: good job! I am approving to merge.

**Review by SaiPriya-WSU — APPROVED** [review]
good job! I am approving to merge.

**Review by Prathyusha-WSU — APPROVED** [review]

**Pull request merged**